src/segments/utils.py

```diff
@@ -302,6 +303,34 @@ def load_release(release: Release) -> Any:
     return json.loads(content.content)
 
 
+def load_release_as_model(release: Union[str, Release]) -> ParsedRelease:
+    """Load a release and return a model.
+
+    Args:
+        release: A Segments release resulting from :meth:`.get_release` or a path to the release file resulting from :meth:`.load_release`.
+    Raises:
+        :exc:`ValueError`: If the release is pending or has failed.
+    """
+    # If it's a file path
+    if isinstance(release, str):
+        with open(release) as release_file:
+            release_dict = json.load(release_file)
+            release = parse_obj_as(Release, release_dict)
+
+    assert isinstance(
```

Comment:
Why this assertion? Is it every possible that it doesn't return a Release object?